nsv13/code/modules/overmap/radar.dm

```diff
@@ -35,16 +37,19 @@
 	var/obj/item/supplypod_beacon/beacon
 	var/sensor_mode = SENSOR_MODE_PASSIVE
 	var/radar_delay = MIN_RADAR_DELAY
+	// Whether we use DRADIS-assisted targeting.
+	var/dradis_targeting = FALSE
+	// Whether we can use the radar
+	var/can_use_radar = TRUE
```

Comment:
Me when the ship's gunner (the fool) used their dradis manually, setting it as the default for the ship and allowing me to ping using it through the viewscreens because the last touched console is set as the ship default. 
Or anyone else uses a different dradis consoles which removes the gunner's (the fool again, but for *not* using the console this time) ability to lock on and ping.

I know this is just a more general problem with overmap ships but I feel it's still worth mentioning possible problems. Could be 'fixed' by giving viewscreens ~~good code~~ an internal dradis and prioritizing that on use.

Reply:
Pain. I'll see if I can work something out.

Reply:
Aight, did some stuff that should fix viewscreen pinging. It looks like there's already something to prevent minor consoles from becoming the "main" console, though.

I can try reworking it further if you want, but that'd probably be too out of scope for this PR.